### DISCUSSION TITLE
npm ERR! code E404

### DISCUSSION THREAD

**Discussion by bryanpaget** [datopian/portal#75] [Q&A]
Hello Portal.js Developers!

I am interested in making a beautiful front-end for my CKAN-powered data catalogue!

When I try ` npm init portal-app my-data-portal` I get a 404 error (see below).

```
(base) bryan@my-computer:~/Projects/ckan$ npm init portal-app my-data-portal

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/create-portal-app - Not found
npm ERR! 404 
npm ERR! 404  'create-portal-app@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/bryan/.npm/_logs/2021-01-22T14_24_04_727Z-debug.log
Install for create-portal-app@latest failed with code 1
```

I am very new to NPM and Javascript, so perhaps I am just doing something wrong. Any assistance would be appreciated!

Thank you!

Bryan P

**Comment by rufuspollock** [comment]
@bryanpaget sorry to miss this when you posted this - we'll look into the error. /cc @risenW

**Reply by rufuspollock** [reply]
@bryanpaget is this something you would still like help with?

**Reply by bryanpaget** [reply]
Hi Rufus. I'm not sure. My team explored CKAN but my developers ended up having a hard time developing with it. They eventually moved to Amundsen... which was underwhelming in terms of features and UI/UX. Now my team is looking into Azure and its data cataloging stuff. :-( That's not my choice. I always want to do open source and I thought Open Knowledge Foundation and Datopian seemed like the kind of organizations I'd like to support but I'm also just one person and most people I work with don't care.

Do you think CKAN and portal.js can still offer something compelling as an addon to Azure? That's something we are still considering: using Azure and its data offerings as a backend but we may still want a custom catalog based on CKAN/portal.js.
